Translations of `Operational Studies` component updated

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OSRD/Operational Studies](https://hosted.weblate.org/projects/osrd/operational-studies/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/osrd/operational-studies/horizontal-auto.svg)

Castavo committed:
front: operational-studies: translated using weblate (Spanish)

Currently translated at 1.1% (6 of 506 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/es/

Signed-off-by: Baptiste Prévot <pro.baptiste.prevot@gmail.com>

SharglutDev commented on front/public/locales/es/operational-studies.json:
```diff
@@ -1 +1,10 @@
-{}
+{
+  "boards": {
+    "conflicts": "conflictos",
```
We should have an uppercase here

clarani replied:
Fixed

SharglutDev commented on front/public/locales/es/operational-studies.json:
```diff
@@ -1 +1,10 @@
+    "macro": "Macro",
+    "map": "Mapa",
+    "sdd": "GEV",
+    "std": "GET",
```
These acronyms should be different in spanish

clarani replied:
Fixed

Commit:
front: operational-studies: add translation using weblate (Polish)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/pl/

Signed-off-by: BigDy HQ <ened12ek23@gmail.com>

Commit:
front: operational-studies: translated using weblate (Polish)

Currently translated at 0.5% (3 of 506 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/pl/

Signed-off-by: BigDy HQ <ened12ek23@gmail.com>

braintrue committed:
front: operational-studies: translated using weblate (Korean)

Currently translated at 0.9% (5 of 514 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/ko/

Signed-off-by: Changseop Yeom <ckdtjqdlgh@gmail.com>

Commit:
front: operational-studies: translated using weblate (German)

Currently translated at 72.3% (372 of 514 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/de/

Signed-off-by: Geolas Play <geosplay@gmail.com>

clarani committed:
front: operational-studies: translated using weblate (Spanish)

Currently translated at 1.1% (6 of 522 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/es/

Signed-off-by: clarani <clara.ni@outlook.fr>

flomonster committed:
front: operational-studies: translated using weblate (English)

Currently translated at 100.0% (522 of 522 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/en/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>

flomonster committed:
front: operational-studies: translated using weblate (French)

Currently translated at 100.0% (522 of 522 strings)

Translation: OSRD/Operational Studies
Translate-URL: https://hosted.weblate.org/projects/osrd/operational-studies/fr/

Signed-off-by: Florian Amsallem <florian.amsallem@gmail.com>
